Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -71,6 +71,8 @@ void JitArm64::Init()
   GenerateAsm();
 
   ResetFreeMemoryRanges();
+
+  m_system.GetJitInterface().UpdateMembase();
```

Comment:
This needs to be done in `Run`, not in `Init`. (Or is there a reason I'm missing why this is also needed in `Init`?)